openapi.yml

```diff
@@ -2980,6 +2980,129 @@ paths:
             operationId: createNotification
             summary: Create notifications
             description: Enqueues notifications to filtered Hosts and/or SignIns via specified channels
+    '/registrations/{registrationId}':
+        summary: Path used to display a single Registration.
+        description: The REST endpoint/path used to get single instances of a `Registration`.
+        get:
+            tags:
+                - Registration
+            responses:
+                '200':
+                    content:
+                        application/json:
+                            schema:
+                                $ref: '#/components/schemas/Registration'
+                            examples:
+                                Default:
+                                    value:
+                                        id: 96
+                                        created_at: '2018-02-10T09:30Z'
+                                        invite_id: 99
+                                        visitor_id: 44
```

Comment:
Sorry. I had heard you say this was for create earlier, so I skipped over these.
If this is for showing/indexing the list of registrations then these need to be `invite` and `visitor` and have the data types of those things.

This will let us side-load them and be consistent with the rest of the API